dolphinscheduler-task-plugin/dolphinscheduler-task-seatunnel/src/main/java/org/apache/dolphinscheduler/plugin/task/seatunnel/EngineEnum.java

```diff
@@ -1,40 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.dolphinscheduler.plugin.task.seatunnel;
-
-public enum EngineEnum {
-
-    FLINK("${SEATUNNEL_HOME}/bin/start-seatunnel-flink.sh"),
-    SPARK("${SEATUNNEL_HOME}/bin/start-seatunnel-spark.sh"),
```

Comment:
delete the enum class due to seatunnel changing its startup script, so currently we have to support

```
seatunnel.sh
start-seatunnel-flink-13-connector-v2.sh
start-seatunnel-flink-15-connector-v2.sh
start-seatunnel-flink-connector-v2.sh
start-seatunnel-flink.sh
start-seatunnel-spark-2-connector-v2.sh
start-seatunnel-spark-3-connector-v2.sh
start-seatunnel-spark-connector-v2.sh
start-seatunnel-spark.sh
```

nine startups in both frontend and backend. Since we do not know whether seatunnel will change their startup script in the future or not. So I think we should remove this enum and just use the startup script name from front-end. please notice we do not ask users manually enter the startup script name because we provide drop-down box for it. Add this just for reducing the code change scope when startup script adding/changing

Reply:
for reviewer, we have a discussion in https://github.com/apache/dolphinscheduler/pull/14131#issuecomment-1551153028

Reply:
The new scream shot look like this
![seatunnel](https://github.com/apache/dolphinscheduler/assets/15820530/194228ce-1c82-4d37-984b-87a5e172d6b5)